edge-apps/menu-board/static/css/style.css

```diff
@@ -0,0 +1,346 @@
+  text-shadow:
+    0 2px 4px rgba(0, 0, 0, 0.4),
+    0 0 30px var(--purple-tint);
+}
+
+.menu-grid {
+  pointer-events: auto;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 1.25rem;
+  padding: 0 1rem;
+  max-width: 1920px;
+  margin: 0 auto;
+  width: 100%;
+  opacity: 1;
+  transition: opacity 0.5s ease;
+}
+
+.menu-grid.fade-out {
+  opacity: 0;
+}
+
+.pager {
+  position: fixed;
+  bottom: 2rem;
+  left: 50%;
+  transform: translateX(-50%);
+  display: flex;
+  gap: 0.5rem;
+  z-index: 10;
+  background: rgba(0, 0, 0, 0.5);
+  padding: 0.5rem;
+  border-radius: 2rem;
+  backdrop-filter: blur(5px);
+}
+
+.page-dot {
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  background: rgba(255, 255, 255, 0.3);
+  transition: all 0.3s ease;
+}
+
+.page-dot.active {
+  background: var(--accent-color);
+  transform: scale(1.2);
+}
+
+.menu-item {
+  background: rgba(8, 8, 24, 0.95);
+  backdrop-filter: blur(10px);
+  border-radius: 12px;
+  padding: 1.25rem;
+  display: grid;
+  grid-template-areas:
+    "title title"
+    "desc price"
+    "labels labels";
+  grid-template-columns: 1fr auto;
+  grid-template-rows: auto 1fr auto;
+  align-items: center;
+  gap: 0.75rem;
+  position: relative;
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.3);
+  transition: all 0.3s ease;
+  min-height: 140px;
+}
+
+.menu-item:hover {
+  border-color: rgba(126, 44, 210, 0.3);
+  box-shadow:
+    0 8px 30px rgba(0, 0, 0, 0.4),
+    0 0 20px rgba(126, 44, 210, 0.2);
+  transform: translateY(-2px);
+}
+
+.menu-item h2 {
+  grid-area: title;
+  font-size: 1.5rem;
+  margin-bottom: 0.25rem;
+  color: var(--accent-color);
+  font-family: "Playfair Display", serif;
+  font-weight: 600;
+  text-shadow: 0 2px 4px rgba(0, 0, 0, 0.4);
+}
+
+.content {
+  grid-area: desc;
+  display: flex;
+  align-items: center;
+}
+
+.menu-item p {
+  font-size: 0.9rem;
+  line-height: 1.4;
+  color: rgba(255, 255, 255, 0.9);
+  margin: 0;
+  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.2);
+}
+
+.price {
+  grid-area: price;
+  font-size: 1.5rem;
+  color: var(--accent-color);
+  font-family: "Playfair Display", serif;
+  font-weight: 600;
+  display: flex;
+  align-items: center;
+  justify-self: end;
+  text-shadow: 0 2px 4px rgba(0, 0, 0, 0.4);
+}
+
+.price::before {
+  content: "$";
+  font-size: 0.8em;
+  margin-right: 0.1em;
+}
+
+.labels {
+  grid-area: labels;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.4rem;
+  margin-top: 0.25rem;
+}
+
+.label {
+  font-size: 0.7rem;
+  padding: 0.2rem 0.5rem;
+  border-radius: 1rem;
+  background: var(--purple-tint);
+  color: var(--accent-color);
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  font-weight: 600;
+}
+
+.label.vegetarian {
+  background: rgba(76, 175, 80, 0.2);
+  color: #4caf50;
+}
+
+.label.spicy {
+  background: rgba(244, 67, 54, 0.2);
+  color: #f44336;
+}
+
+.label.gluten-free {
+  background: rgba(33, 150, 243, 0.2);
+  color: #2196f3;
+}
```

Comment:
Probably need to support more. Depends on where the screen is being used though.

Reply:
Thanks, @iamplugged. We can add more styles for specific options later.
By default, any other labels will have a purple background color:

<img width="1859" height="957" alt="Image" src="https://github.com/user-attachments/assets/3f858a65-82ea-4963-bb19-e641de04acc0" />